api/v1alpha1/virtualmachine_types.go

```diff
@@ -166,6 +167,11 @@ type Probe struct {
 	// Defaults to 10 seconds. Minimum value is 1.
 	// +optional
 	TimeoutSeconds int32 `json:"timeoutSeconds,omitempty"`
```

Comment:
Part implementation detail, and also that these probes are not execute via anything similar to kubelet, I think we need to either define a maximum here, or internally cap the provided timeout.

Reply:
Yeah we can define a maximum here. Is 30 seconds a reasonable value for the maximum timeout?